### ISSUE TITLE
Citation

### ISSUE THREAD

**Issue by dmetivie** [SciML/QuasiMonteCarlo.jl#90]
I am finishing revising a paper (see [v1](https://hal.science/hal-03631879/document)) where I use `QuasiMonteCarlo.jl`. 
I read about how to cite Julia packages ([here](https://discourse.julialang.org/t/how-to-cite-julia-packages-for-academic-purpose/80124)).
What is the common practice for SciML packages (guess answer is [here](https://sciml.ai/citing/))?
As there are no paper yet, can something that work?
  >@misc{2019_QMC,
    title = {QuasiMonteCarlo.jl},
    author = {Various contributors},
    year = {2019},
    howpublished = {\url{https://github.com/SciML/QuasiMonteCarlo.jl}}
  }

BTW, any paper plans? Since 1/4 of the code comes from various packages (Sobol, Lattice, Latin, I am not sure if this is a legit idea or not,

**Comment by ChrisRackauckas** [comment]
Yeah something like that would work for now. I think down the line we may want to go for a JOSS paper. I don't think we have enough to mathematically say to go for a full 20 pager, but there is merit here to JOSS.
